linera-views/tests/operations_tests.rs

```diff
@@ -139,5 +204,110 @@ async fn test_ordering_memory_specific() {
         (vec![0, 2], Vec::new()),
         (vec![0, 2, 0], Vec::new()),
     ];
-    test_ordering_keys_key_value_vec(key_value_operation, key_value_vec).await;
+    test_readings_vec(key_value_operation, key_value_vec).await;
+}
+
+#[cfg(test)]
+async fn test_writings_random<OP: KeyValueStoreClient + Sync>(key_value_operation: OP) {
+    let mut rng = rand::rngs::StdRng::seed_from_u64(2);
+    let mut kv_state = BTreeMap::new();
+    let n_oper = 100;
+    let siz_batch = 8;
+    // key space has size 4^4 = 256 so we necessarily encounter collisions
+    let key_prefix = vec![0];
+    for _ in 0..n_oper {
+        let mut batch = Batch::new();
+        for _ in 0..siz_batch {
+            let thr = rng.gen_range(0..8);
```

Comment:
What does `thr` stand for?

Reply:
To me, it was `threshold` but maybe `choice` is better.